src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  display: flex;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  background-color: #d4e2e6;
+  border: 1vmin #000 dotted;
+  border-radius: 50%;
+  position: relative;
+
+  &-minute {
+    position: absolute;
+    transform-origin: bottom;
+    bottom: 50%;
```

Comment:
You can make separate selector for such properties to remove doublicated properties